backend/services/theme_service.py

```
@@ -393,25 +393,43 @@ async def _generate_with_runware(
+- DRAMATIC COLOR CHANGE: Make clothing color transformation very obvious and complete
 - NO BUSINESS ATTIRE: Remove suit, tie, office clothing completely
-- SPIRITUAL TRANSFORMATION: Full conversion to spiritual master appearance
+- NO WRONG COLORS: Only use colors specifically mentioned in the description
+- SPIRITUAL TRANSFORMATION: Full conversion to spiritual master appearance with correct daily colors
 
 TECHNICAL SPECS: Photorealistic, high resolution, professional photography, cinematic lighting, masterpiece quality."""
 
-            # 🔒 FACE LOCK + TRANSFORMATION NEGATIVE PROMPT
-            negative_prompt = """different face, changed face, new face, altered face, face swap, face replacement, 
+            # 🎨 DAILY COLOR NEGATIVE PROMPT: Prevent wrong colors for each day
+            day_of_week = datetime.now().weekday() if theme_day is None else theme_day
```

Ruff (0.12.2)

Undefined name `theme_day`

(F821)